### ISSUE TITLE
Install fails on windows

### ISSUE THREAD

**Issue by Page-** [resin-io-modules/drivelist#217]
```
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
```
Is it possible to use node-pre-gyp to install pre-built binaries where possible avoiding issues where build tools are not available

**Comment by jviotti** [comment]
Python is a requirement of `node-gyp` itself (see https://github.com/nodejs/node-gyp#on-windows). You can alternatively install `npm install --global --production windows-build-tools`. Would that work for you?

**Comment by Page-** [comment]
@jviotti that's why I suggested node-pre-gyp, which can avoid the need for python and other build tools by making use of prebuilt binaries when available and only falling back to node-gyp if there aren't any prebuilt binaries for the current os/arch combo

**Comment by jhermsmeier** [comment]
Closing this, as this has been fixed by #219 and #223
